syft/pkg/cataloger/python/parse_wheel_egg_record.go

```diff
@@ -59,3 +61,35 @@ func parseWheelOrEggRecord(reader io.Reader) ([]pkg.PythonFileRecord, error) {
 
 	return records, nil
 }
+
+func parseInstalledFiles(reader io.Reader, location, sitePackagesRootPath string) ([]pkg.PythonFileRecord, error) {
+	var installedFiles []pkg.PythonFileRecord
+	r := bufio.NewReader(reader)
+	// scanner := bufio.NewScanner(file)
+
+	for {
+		line, err := r.ReadString('\n')
+		if err == io.EOF {
+			break
+		}
+		if err != nil {
```

Comment:
Should failure reading the `installed-files` be a critical error? The caller seems to treat it as critical, but as it wasn't even being done before maybe these should instead be warnings?